pkg/api/api.go

```diff
@@ -322,6 +337,56 @@ type ExpireCodeResponse struct {
 	ErrorCode string `json:"errorCode,omitempty"`
 }
 
+// UserReportRequest defines the structure for a user initiated report.
+// This is a device API hosted on the apiserver.
+//
+// Support level: ALPHA
+// This API is not guaranteed to be backwards compatible until such time it is moved
+// to beta.
+//
+// Inclusion of the test date is required, inclusion of the symptom date is optional.
+//
+// A phone number is required. The verification code is only presented over SMS.
+//
+// The nonce field must be a 128 random bytes, base64 encoded.
+// This nonce field must be passed back on the VerifyCodeRequest request later.
+//
+// Requires API key in a HTTP header, X-API-Key: APIKEY
+type UserReportRequest struct {
+	Padding Padding `json:"padding"`
+
+	SymptomDate string `json:"symptomDate"` // ISO 8601 formatted date, YYYY-MM-DD
```

Comment:
Should we just embed the other struct instead of copying all the fields?

Reply:
it doesn't include all of the fields (and shouldn't)